lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf

```diff
@@ -636,10 +639,11 @@ emqx_ee_bridge_kafka {
                 "<code>none</code> Uses the key from the Kafka message unchanged."
                 "  Note: in this case, then the key must be a valid UTF-8 string.\n"
                 "<code>base64</code> Uses base-64 encoding on the received key."
-            zh: "定义了在通过MQTT转发之前如何处理Kafka消息的 Key。"
-                "<code>none</code> 使用Kafka消息中的 Key 原始值，不进行编码。"
-                "  注意：在这种情况下，Key 必须是一个有效的UTF-8字符串。\n"
-                "<code>base64</code> 对收到的密钥或值使用base-64编码。"
+            zh: "桥接到 EMQX 之前如何处理 Kafka 消息的键或值。\n"
```

Comment:
I think the product team used an old version of this doc when the key and value descriptions were the same.  Please help fix "key" and "value" usages here and below.  :pray: